include/ByteBuffer.hpp

```diff
@@ -1,6 +1,7 @@
 #ifndef BYTE_BUFFER_HPP
 #define BYTE_BUFFER_HPP
 
+#include <cstdint>
```

Comment:
Need this for std::byte. When I switched to `gcc` it was complaining about this.